docs/components/framework_components/skpagerwithtabs.md

```diff
@@ -0,0 +1,87 @@
+# SKPagerWithTabs
+
+As [SKPager](skpager.md), show a [ViewPager2](https://developer.android.com/jetpack/androidx/releases/viewpager2). With this component, you can show and customize tab indicator.
+
+## Usage
+In your ScreenVC, declare a SKPagerWithTabs. You also have to declare screens you want to display in `SKUses` annotation :
+
+```kotlin
+@SKUses([MyCardTabVC::class])
+interface MyCardInfoScreenVC : SKScreenVC {
+    val pager: SKPagerWithTabsVC
+}
+```
+Launch the generation with the SKGenerate gradle task.
+In your screen, you have to instanciate your component as below :
+
+```kotlin
+override val pager = SKPagerWithTabs(onUserSwipeToPage = onPageSwiped)
+```
+
+In your xml layout file, whereas with [SKPager](skpager.md) you have to set an id on ViewPager2 widget, on SKPagerWithTabs, the id 
+`pager` must be set on ViewGroup (ConstraintLayout, LinearLayout or whatever you want). It must contain a ViewPager2 and a TabLayout with tag `sk_view_pager2` and `sk_tab_layout`
+
+
+```xml
+<LinearLayout
+    android:id="@+id/pager"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    android:layout_marginTop="10dp"
+    android:gravity="center_horizontal"
+    android:orientation="vertical"
+    app:layout_constraintTop_toBottomOf="@+id/closeButton">
+
+    <com.google.android.material.tabs.TabLayout
+        style="@style/TabStyle"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:tag="sk_tab_layout"
+        app:tabMode="auto" />
+
+    <View
+        android:id="@+id/separator"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="@color/stroke" />
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:tag="sk_view_pager2" />
+</LinearLayout>
+```
+
+`initialPages` could be an empty list. You can set TabPage to display after SKPagerWithTabs init, setting `pages` properties.
```

Comment:
The `initialPages` property can be an empty list. You can define the list of TabPages to be displayed at any time by setting the pages property.